test/NuGet.Core.Tests/NuGet.Common.Test/LocalizedTestCollection.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Globalization;
+using Xunit;
+
+namespace NuGet.Common.Test
+{
+    [CollectionDefinition(TestName, DisableParallelization = true)]
+    public class LocalizedTestCollection
```

Comment:
I think this class can be `static` because all its members are `static`. Also, can this class itself be `internal` because it is only used with in the assembly?

Reply:
Thanks! Fixed.